docs/en/complete_vehicles_mc/modalai_starling.md

```diff
@@ -1,15 +1,17 @@
-# VOXL 2 Starling PX4 Development Drone
+# ModalAI Starling (PX4 Autonomy Developer Kit)
 
-The [Starling](https://modalai.com/starling) is a SLAM development drone supercharged by [VOXL 2](../flight_controller/modalai_voxl_2.md) and PX4 with SWAP-optimized sensors and payloads optimized for indoor and outdoor autonomous navigation.
+The [Starlings](https://www.modalai.com/pages/starlings) are SLAM development drones supercharged by [VOXL 2](../flight_controller/modalai_voxl_2.md) and PX4 with SWAP-optimized sensors and payloads optimized for indoor and outdoor autonomous navigation.
 Powered by Blue UAS Framework autopilot, VOXL 2, the Starling weighs only 275g and boasts an impressive 30 minutes of autonomous indoor flight time.
 
 ![Overview](../../assets/hardware/complete_vehicles/modalai_starling/starling_front_hero.jpg)
 
-The VOXL 2 Starling is a PX4 development drone that houses a [VOXL 2](../flight_controller/modalai_voxl_2.md) companion computer and PX4 flight controller, image sensors, GPS, and connectivity modem and is ready-to-fly out-of-the-box.
-The Starling features ModalAI’s [open SDK](https://docs.modalai.com/voxl-developer-bootcamp/) that has pre-configured autonomy models for computer vision assisted flight.
+The ModelAI PX4 Autonomy Developer Kit is a Starting-based development drone.
```

Comment:
This should be ModalAI, not ModelAI. And "is a Starting-based" should be "is a Starling-based".

Reply:
Thanks @katzfey !! I rushed this.

FYI You can make specific suggestions exactly how you like using the suggestion button

<img width="766" height="206" alt="image" src="https://github.com/user-attachments/assets/2e90f556-b4ad-415e-a41a-86701a6ac7ca" />

```suggestion
The ModalAI PX4 Autonomy Developer Kit is a Starling-based development drone.
```